### PR TITLE
Ernst add close to cards

### PR DESCRIPTION
**Pull request by ernstkui**
Instead of spatial.here, spatial.points and spatial.region the state keeps track of selected assets or if the selection is not an asset, a selected geometry.

DataService reacts to changes in those lists, the layergroups, and temporal state, to get all relevant data for those lists in DataService.assets and DataService.geometries.

Omnibox binds the scope to DataService and ng-repeat and the template-directives do the rendering in the cards. The cards remove the asset/geometry from the state.selected when closed and keep track of the click feedback.

Conceptual changes:
- All cards can be removed from box and map in one click.
- Cards are grouped per asset/geometry and have a header.
- An asset with line geometry is treated as a line.
- Possible to show one line and multiple points.
- Cards with no data show a nodata message
- Cards animate in and out
- Data loops only one level deep. So layergroups with multiple data-layers are no longer supported...
- Events are temporarily out.
- Bolletje is also collateral damage.
- Rain and area is still a little wonky
- 100% bug free.

TODO:
- [ ] Empty utfGrid
- [ ] Update configuration to use single layers per layergroup
- [ ] Check what happens with line geometries that are not straight a to b.
- [ ] Fix rain and area
- [ ] Do for events what this did for assets and geometries
- [ ] Recreate bolletje 
- [ ] annotations
- [ ] csv export
- [ ] dashboard and switch
- [ ] clickfeedback shows small circle or nothing

### CODE REVIEW

File: app/components/data-menu/services/data-service.js [hunk 1]
```diff
@@ -63,7 +66,23 @@ angular.module('data-menu')
       this._createLayerGroups = function (serverSideLayerGroups) {
         var layerGroups = {};
         angular.forEach(serverSideLayerGroups, function (sslg) {
-          this.createLayerGroup(sslg);
+
+          var utf = false;
```
Comment (fritzvd) [inline app/components/data-menu/services/data-service.js]: misschien nog handig om in de volgende pull request deze variable nog te hernoemen naar `utfGrid` aangezien utf andere sentimenten bij mensen oproept. (Denken ze eerder aan de utf8 encoding)

File: app/components/omnibox/controllers/region-controller.js [hunk 2]
```diff
@@ -1,156 +0,0 @@
...
-
-    /**
-     * Update box.content when time changed.
-     */
-    $scope.$watch(State.toString('temporal.timelineMoving'), function (n, o) {
-      if (!State.temporal.timelineMoving) {
-        region = State.spatial.region;
-        if (region && region.properties && region.properties.name) {
-          fillRegion(State.spatial.region);
-        }
```
Comment (fritzvd) [inline app/components/omnibox/controllers/region-controller.js]: https://www.youtube.com/watch?v=7ky8yzS4VZY

File: app/components/omnibox/directives/omnibox-directives.js [hunk 3]
```diff
@@ -18,53 +18,53 @@ angular.module("omnibox")
...
-        if (n === o) { return true; }
-        finalizeTemplateRendering();
-      });
+      // scope.$watch(State.toString('context'), function (n, o) {
+      //   if (n === o) { return true; }
+      //   finalizeTemplateRendering();
+      // });
 
-      finalizeTemplateRendering();
+      // finalizeTemplateRendering();
```
Comment (fritzvd) [inline app/components/omnibox/directives/omnibox-directives.js]: kunst

File: app/components/omnibox/controllers/region-controller.js (hunk 2 above)
Comment (ernstkui) [inline app/components/omnibox/controllers/region-controller.js]: Ik kom er niet direct achter wat voor actiepunten ik hier uit kan halen

File: app/components/omnibox/directives/omnibox-directives.js (hunk 3 above)
Comment (ernstkui) [inline app/components/omnibox/directives/omnibox-directives.js]: Hier moet ik nog wat mee. Die scenarios moeten nog kunnen switchen...

File: app/components/data-menu/services/asset-service.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+          .then(function (response) {
+            response.data.entity_name = entity;
+            return response.data;
+          });
+        }
+
+        else {
+          var defer = $q.defer();
+          defer.resolve();
+          return defer.promise;
```
Comment (fritzvd) [inline app/components/data-menu/services/asset-service.js]: valt makkelijk te testen.. doen dus

**Pull request closed without merging**